compiler/tests-common/tests/org/jetbrains/kotlin/cli/AbstractCliTest.java

```diff
@@ -98,9 +100,21 @@ public static String getNormalizedCompilerOutput(@NotNull String pureOutput, @Nu
                 .replace("\n" + Usage.BAT_DELIMITER_CHARACTERS_NOTE + "\n", "")
                 .replaceAll("log4j:WARN.*\n", "");
 
+        normalizedOutputWithoutExitCode = replaceLanguageVersion(normalizedOutputWithoutExitCode);
+
         return exitCode == null ? normalizedOutputWithoutExitCode : (normalizedOutputWithoutExitCode + exitCode + "\n");
     }
 
+    @NotNull
+    private static String replaceLanguageVersion(@NotNull String compilerOutput) {
+        if (compilerOutput.isEmpty() || LanguageVersion.FIRST_NON_STABLE == null) {
+            return compilerOutput;
+        }
+
+        JvmMetadataVersion mv = GenerationState.Companion.getLANGUAGE_TO_METADATA_VERSION().get(LanguageVersion.FIRST_NON_STABLE);
+        return compilerOutput.replace(mv.toString(), "$LANGUAGE_VERSION$");
```

Comment:
Since you're only replacing very specific version, I'd prefer this token to be something like `$FIRST_NON_STABLE$`